trieste/models/gpflow/sampler.py

```diff
@@ -44,6 +44,28 @@
 )
 
 
+def qmc_normal_samples(batch_shape: tf.TensorShape, n_sample_dim: int, skip: int = 0) -> tf.Tensor:
+    """
+    Generates a batch of shape `batch_shape` sobol samples, skipping the first `skip`, where each
+    sample has dimension `n_sample_dim`.
+    """
+    sobol_samples = tf.math.sobol_sample(
+        dim=n_sample_dim,
+        num_results=tf.reduce_prod(batch_shape),
```

Comment:
Do we have any reason now to allow for a batch_shape that is not just a single value? in the batch case you do the product outside anyway and in the independent case batch_shape is a single integer.